File: Desktop/DSA/T93/16.py
#subarray in a range 

def main(array:list[int],x:int,y:int)->int:
    
    count=0
    n=len(array)

    for i in range(n):
        
        maximum = array[i]

        for j in range(i,n):
            
            maximum=max(maximum,array[j])

            if x<=maximum<=y:
                
                count += 1
                
    return count

arr = [1, 2, 3, 4, 5]
X = 2
Y = 4
print(main(arr,X,Y))

File: Desktop/DSA/T93/16.js

//bubble sort


const main=(array) => {

    let n=array.length

    for (let i=0;i <n ;i ++){

        for (let j=0;j < n-i-1 ;j ++){

            if  (array[j]>array[j+1]){

                [array[j],array[j+1]]=[array[j+1],array[j]]
            }
        }
    }

    return array
}

const arr = [6, 8, 9, 2, 3];
console.log(main(arr));